frontend/__tests__/sum.test.ts

```diff
@@ -1,8 +1,19 @@
 import { sum } from "../src/sum"
-import { describe, test, expect } from "@jest/globals"
+import { describe, test, expect, beforeAll } from "@jest/globals"
+import { server } from '../src/mocks/node.ts';
 
 describe('Sum function', () => {
     test('Returns correct value', () => {
         expect(sum(2, 3)).toEqual(5)
     })
+})
+beforeAll(() => {
```

Comment:
Try moving the beforeAll above the describe block on line 5. This will make it skip that test.

Reply:
done. Thanks for your help.